css/styles.css

```diff
@@ -318,26 +376,77 @@ p {
 
 .phase-items {    
     list-style: none;
-    font-family: Poppins, Helvetica, sans-serif;
     font-size: 18px;
-    font-weight: 400;
     line-height: 28px;
     letter-spacing: 0;
-    text-align: left;
     color: #c2c3c5;
+    text-transform: capitalizeupp;
+}
+
+.phase-items>p::before {
```

Comment:
We need to add class attributes to all tags, and then use classes for styling. For example:

.phase-items>p::before ==> .phase-text::before